newrelic/resource_newrelic_entity_tags.go

```diff
@@ -193,6 +204,20 @@ func resourceNewRelicEntityTagsUpdate(ctx context.Context, d *schema.ResourceDat
 	return nil
 }
 
+// This is needed until the client implements a GetTags method with the same
+// tag type as the rest of the methods.
```

Comment:
Is there an issue for doing this? :)

Reply:
https://github.com/newrelic/newrelic-client-go/issues/747